ocdaction/challenges/views.py

```diff
@@ -160,12 +167,24 @@ def challenge_score_form_new(request, challenge_id):
             challenge.save()
             anxiety_score_card.save()
 
-            return redirect(challenge_score_form, challenge_id=challenge_id, score_id=anxiety_score_card.id)
+            return redirect(
+                challenge_score_form,
+                challenge_id=challenge_id,
+                score_id=anxiety_score_card.id
+            )
```

Comment:
If it's a formatting changing (which is by the way a very subjective change in this context) can we please do it as a separate commit next time, as it's hard to read if any other changes weren't introduce here

Reply:
Sure will do, I am using the standard flake8 for python